src/cosalib/meta.py

```diff
@@ -36,55 +67,15 @@ def read(self):
         self.clear()
         # Load the file
         self.update(load_json(self._meta_path))
+        self.validate()
 
     def write(self):
         """
         Write out the dict to the meta path.
         """
+        self.validate()
         write_json(self._meta_path, dict(self))
 
-    def get(self, *args):
```

Comment:
Why is this code being deleted?

Reply:
`GenericBuildMeta(dict)` already provides `get`, and `set`.  The behavior of `dict.get()` was changed and it was causing errors.

Reply:
I added 'em back.